core/trino-main/src/test/java/io/trino/sql/analyzer/TestAnalyzer.java

```diff
@@ -4032,6 +4032,12 @@ public void testInValidJoinOnClause()
                 .hasMessage("line 1:69: JOIN ON clause must evaluate to a boolean: actual type row(boolean, boolean)");
     }
 
+    @Test
+    public void testValidAggregationFilter()
```

Comment:
This name is too vague. Change it to `testNullAggregationFilter`

Reply:
Fixed.